### ISSUE TITLE
Deter or prevent setting up alerts on old request threads

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#5952]
It appears that confused WhatDoTheyKnow.com users are setting up alerts to follow old request threads on which further correspondence or annotations would not be expected. 

It might be useful to have an intermediate stage warning users requests are old and unlikely to have further activity, or to simply prevent signing up for alerts on request pages which are closed to new correspondence. 

There is a chance an old request could become of interest in the future and have annotations added years later and ideally we would support following to keep up with any such threads. 

The problem appears to be users signing up for alerts unintentionally. The tracks in question often relate to personal transactions with government. 

Users setting up these alerts often write to us in response to their "Confirm you want to follow the request" email, and they appear usually to be confused users who are struggling to understand or use our service, or those who are actually seeking something other than our service. 

Related: #4884 "Deter accidental subscription to alerts for all successful requests"

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
